packages/services/src/terminal/terminal.ts

```diff
@@ -19,9 +19,10 @@ export { IModel, isAvailable };
 export namespace ITerminal {
   export interface IOptions {
     /**
-     * Terminal name.
+     * Terminal name and directory it will start in.
      */
-    name: string;
+    name?: string;
+    cwd?: string | undefined;
```

Comment:
you are right, my comment https://github.com/jupyterlab/jupyterlab/pull/12250#discussion_r831202428 is wrong... the problem, then, is that this PR doesn't work when starting notebook in the root folder, because the request we send to terminado has issues with `''`  as cwd. There are several places where this can be fixed, but maybe the easiest is to go back to my original test `if (cwd)`?

Reply:
Thanks a lot for pushing the test through @rccern. Would you mind adding the above suggestion? After that we are good to go.